site/homepage/content/getting-started/index.md

```diff
@@ -121,46 +121,50 @@ Before a device can connect to Hono and publish any data, the corresponding info
 
 ### Creating a new Tenant
 
-Choose a (random) tenant identifier and register it using the Device Registry's HTTP API (replace `my-tenant` with your identifier):
+Register a tenant it using the Device Registry's management HTTP API (a random tenant identifier will be generated):
 ~~~sh
-export MY_TENANT=my-tenant
-curl -i -H "content-type: application/json" --data-binary '{"tenant-id": "'$MY_TENANT'"}' http://$REGISTRY_IP:28080/tenant
+curl -i -H "content-type: application/json" --data-binary '{}' http://$REGISTRY_IP:28080/tenants
 
 HTTP/1.1 201 Created
-Location:  /tenant/my-tenant
-Content-Length: 0
-~~~
+etag: resource_version_value
```

Comment:
are you sure that this is lower case?

Reply:
the curl output is lowercase

Reply:
that is strange, since it is defined as *ETag* in RFC2616.

Reply:
Section 4.2 about headers explicitly mention the headers names to be case-insensitive. So this is not an issue IMHO. Also, it's better to keep the _curl_ output as _verbatim_ as possible i think.

https://tools.ietf.org/html/rfc2616#section-4.2

Reply:
No it is not an issue, I was just wondering why the *etag* header shows up in lower case while the *Location* header shows up starting with an upper case letter ...

Reply:
I think we should use the *real* values generated by the system here and in the *location* header and then add a note below describing that the values that the user will get will be different ones.

Reply:
I updated it. I suggested in the note to export it as en environment variable and added the command to do it in the note. I think it make the two steps clearer.